dash-pipeline/bmv2/stages/outbound_routing.p4

```diff
@@ -33,8 +50,13 @@ control outbound_routing_stage(inout headers_t hdr,
             return;
         }
 
+        routing_group.apply();
+        if (meta.eni_data.routing_group_admin_state) {
```

Comment:
maybe it is better to create a dedicated structure for routing group data, then we can use meta.routing_group_data.admin_state to access the related states. 

As metadata becomes more and more complicated, it will be more future-proof.

Reply:
Done